games/f.yaml

```diff
@@ -176,6 +176,30 @@
   url: http://www.newbreedsoftware.com/fop/
   info: A Dandy clone with Gauntlet influences. Choose from 8 classes and roam the maze taking treasure, vanquishing monters, and finding the exit. Use explosives against enemies and keys on locked doors.
 
+- name: Fish Fillets NG
+  originals:
+  - Fish Fillets
+  type: remake
+  repo: 'https://sourceforge.net/projects/fillets/'
+  url: 'http://fillets.sourceforge.net/'
+  development: complete
+  status: playable
+  content: free
+  lang:
+  - C++
+  - Lua
+  framework:
+  - SDL
+  license:
+  - GPL2
+  info: Cross-platform port of the original game sources.
+  updated: 2022-05-05
+  images:
+  - 'http://fillets.sourceforge.net/img/screenshot/ffng-pyramid.png'
+  - 'http://fillets.sourceforge.net/img/screenshot/ffng-chest.png'
```

Comment:
images should be HTTPS otherwise they don't show when embedded

Reply:
Unfortunately, the SourceForge project site does not support HTTPS so I hade to replace both images with one from Wikipedia.